source/channels/message-priority.rst

```diff
@@ -42,4 +62,7 @@ When you request acknowlegement of a message, an **Acknowledge** |acknowledge-bu
 .. tip::
 
     - When you have push notifications enabled on mobile, you'll be notified every five minutes until you acknowledge or reply to the message.
-    - After acknowledging a message, you have up to five minutes to change your mind. Select the **Acknowledged** |acknowledged-icon| button again to remove your name from the list of acknowledged users.
\ No newline at end of file
+    - After acknowledging a message, you have up to five minutes to change your mind. Select the **Acknowledged** |acknowledged-icon| button again to remove your name from the list of acknowledged users.
```

Comment:
@esethna I think we have discussed it, but just to double check here; "Unacknowledging" doesn't re-start the notifications, once stopped.

Basically, once notifications are stopped by _any_ action, notifications won't restart by _undoing_ the action.

Reply:
+1, makes sense